src/ui/pages/index.jsx

```diff
@@ -1,11 +1,37 @@
 import React from 'react';
-import {Flex} from '@chakra-ui/react';
+import {useSession, signOut} from 'next-auth/react';
+import {Button, Flex} from '@chakra-ui/react';
 import PropTypes from 'prop-types';
 import PageWrapper from '../components/common/PageWrapper';
 import WhyJobTracker from '../components/landing_page/WhyJobTracker';
 import LoginContainer from '../components/landing_page/LoginContainer';
+import LoadingSpinner from '../components/common/LoadingSpinner';
 
 export default function Home() {
+  const {data: session, status} = useSession();
+
+  if (status === 'loading') return <LoadingSpinner />;
+
+  /* Dashboard for Logged In Users */
+  if (session) {
+    /* TODO: Create Dashboard -- Displaying Session Tokens Below for Debugging*/
+    return (
+      <Flex direction="column" p={16} gap={16}>
+        <div>
+          <strong>accessToken:</strong> {session.accessToken}
+        </div>
+        <div>
+          <strong>expires:</strong> {session.expires}
```

Comment:
Does Google control expiration?

Reply:
It looks like next-auth controls the expiration based on "idle" time. We can determine how long to set the expiration per the docs: https://next-auth.js.org/configuration/options#session . The default expiration is 30 days.

We can also use a callback to extend the token after expiration: https://next-auth.js.org/tutorials/refresh-token-rotation